subsys/bluetooth/audio/has_client.c

```diff
@@ -1000,3 +1000,24 @@ static void disconnected(struct bt_conn *conn, uint8_t reason)
 BT_CONN_CB_DEFINE(conn_cb) = {
 	.disconnected = disconnected,
 };
+
+#if defined(CONFIG_BT_TESTING)
+int bt_has_client_test_info_get(struct bt_conn *conn, struct bt_has_client_test_info *info)
+{
+	struct has_inst *inst;
+
+	__ASSERT_NO_MSG(conn != NULL);
+	__ASSERT_NO_MSG(info != NULL);
+
+	inst = inst_by_conn(conn);
+	if (inst == NULL) {
+		return -ENODEV;
+	}
+
+	info->has_hearing_aid_features_handle = inst->features_subscription.value_handle;
+	info->has_control_point_handle = inst->control_point_subscription.value_handle;
+	info->has_active_preset_index_handle = inst->active_index_subscription.value_handle;
+
+	return 0;
+}
+#endif /* CONFIG_BT_TESTING */
```

Comment:
What we have done for the other implementations is to move the definition of e.g. `struct has_inst` to the internal header files, and simply included them in the tests. 

This avoids having test specific functions in the implementations. 

Did you consider this approach, and if so, why did you go with this approach?

Reply:
No, I did not consider this, but it's a better approach actually. Will do that.